tests/testthat/resources/file exists.Rmd

```diff
@@ -0,0 +1,6 @@
+
+```{r}
+if (!file.exists("empty.csv"))
+   stop("empty.csv expected in ", getwd())
+```
+
```

Comment:
Done

On Wed, Oct 21, 2015 at 5:08 PM, Yihui Xie <notifications@github.com> wrote:

> In tests/testthat/resources/file exists.Rmd
> <https://github.com/rstudio/rmarkdown/pull/534#discussion_r42682559>:
>
> > @@ -0,0 +1,6 @@
> > +
> > +```{r}
> > +if (!file.exists("empty.csv"))
> > +   stop("empty.csv expected in ", getwd())
> > +```
> > +
>
> This filename will trigger a warning in R CMD check. I'm surprised that
> we missed it because we didn't enable Travis CI for this repo:
> https://travis-ci.org/profile/rstudio Can someone with admin privilege
> enable it? @jjallaire <https://github.com/jjallaire>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rmarkdown/pull/534/files#r42682559>.
>